<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QMainWindow" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlPanel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="current_connection_label">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>DISCONNECT</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="data_time_edit">
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::Section::YearSection</enum>
          </property>
          <property name="displayFormat">
           <string>yyyy/MM/dd H:mm:ss</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="open_n2o_fill_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>N2O_FILL_OPEN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close_n2o_fill_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>N2O_FILL_CLOSE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="n2o_fill_label">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>OPEN</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="open_n2o_dump_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>N2O_DUMP_OPEN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close_n2o_dump_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>N2O_DUMP_CLOSE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="n2o_dump_label">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>OPEN</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="open_o2_fill_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>O2_FILL_OPEN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close_o2_fill_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>O2_FILL_CLOSE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="o2_fill_label">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>OPEN</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="on_ignition_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>IGNITION_ON</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="off_ignition_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>IGNITION_OFF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ignition_label">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>OPEN</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="action_socket_settings"/>
    <addaction name="action_connect"/>
    <addaction name="action_disconnect"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about_qt"/>
    <addaction name="action_help"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_socket_settings">
   <property name="text">
    <string>TCP/IP</string>
   </property>
  </action>
  <action name="action_connect">
   <property name="text">
    <string>connect</string>
   </property>
  </action>
  <action name="action_disconnect">
   <property name="text">
    <string>disconnect</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
